client/src/modules/privacy-policy/components/Policy.jsx

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable react/no-danger */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { FormattedMessage } from 'react-intl';
+
+import { getCurrentUser } from 'modules/user/model/selectors';
+import { UserPropType } from 'common/constants/propTypes';
+import './Policy.scss';
+
+const Policy = ({ currentUser, dangerouslySetInnerHTML }) => (
+  <>
+    <div className="policy" dangerouslySetInnerHTML={dangerouslySetInnerHTML} />
```

Comment:
I installed `marked` and `DOMPurify` libraries instead of `Webpack` loaders I used before and the result I got was exactly the same. So I leave them in the app because of `DOMPurify` which gives us additional protection. 

Regardless, I can't do anything like this:
```
<div>{resultOfMarkedAndDOMPurifyWork}</dv>
```
because the result I get is the type of string, not JSX or HTML. Even in the article, in the author's code there is:
```
<span dangerouslySetInnerHTML={this.rawMarkup()} />
```
Maybe I don't understand something but  I think that usage of [dangerouslySetInnerHTML](https://reactjs.org/docs/dom-elements.html#dangerouslysetinnerhtml) is necessary because 

> dangerouslySetInnerHTML is React’s replacement for using innerHTML in the browser DOM. In general, setting HTML from code is risky because it’s easy to inadvertently expose your users to a cross-site scripting (XSS) attack. So, you can set HTML directly from React, but you have to type out dangerouslySetInnerHTML and pass an object with a `__html`key, to remind yourself that it’s dangerous:

But the string I try to display comes from our own file, not from the user's input and additionally it is sanitized now.

Reply:
As React documentation says,  `dangerouslySetInnerHTML` prop is named to warn the developer that he is trying to innerHTML to the DOM which may cause some XSS attack if the HTML markup is coming from third-parties. 

So  `dangerouslySetInnerHTML` should be used with some HTML sanitizer like DOMPurify.
So the correct example is: 
`<div  dangerouslySetInnerHTML={DOMPurify.sanitize('...danger HTML goes here'); />`